scripts/mocha.js

```diff
@@ -0,0 +1,30 @@
+const Mocha = require('mocha');
+const path = require('path');
+const express = require('express');
+const serveStatic = require('serve-static');
+const fs = require('fs');
+
+if (!fs.existsSync(path.resolve(__dirname, '../dist'))) {
+  // Do something
+  process.exit(1);
```

Comment:
I was just going through and saw this termination. Maybe a more helpful warning or message will be closer to "do something" :)